CHANGELOG.md

```diff
@@ -2,10 +2,7 @@
 
 ## [Unreleased]
 
-- Remove an import warning in case the import statement points to a local unnamed module.
-- Discover the correct remote to use for fetching dependencies in the `mod update` and `mod prune`
-  commands, in case it's an unnamed module and there are a mix of remotes in the `buf.yaml`
-  configuration file.
```

Comment:
@bufdev I still think we should keep a changelog entry for this. `buf mod update` and `buf mod prune` behavior changed, and now we select the remote for dependencies more intuitively, allowing pro customers to keep local modules unnamed.